demisto_sdk/commands/common/tests/pack_unique_files_test.py

```diff
@@ -175,6 +177,32 @@ def error_raising_function(argument):
         assert not self.validator.validate_pack_dependencies()
         assert Errors.invalid_id_set()[0] in self.validator.get_errors()
 
+    def test_validate_core_pack_dependencies(self, mocker):
```

Comment:
as an extension of my comment in the validation itself - please change this test to include multiple non-core dependencies and that the error prints them all.